db/models.py

```diff
@@ -50,3 +53,65 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(to="User",
+                             on_delete=models.CASCADE,
+                             null=True,
+                             blank=True,
+                             related_name="orders")
+
+    def __str__(self) -> str:
+        return str(self.created_at)
+
+    class Meta:
+        ordering = ("-created_at",)
+
+
+class Ticket(models.Model):
+    movie_session = models.ForeignKey(to=MovieSession,
+                                      on_delete=models.CASCADE)
+    order = models.ForeignKey(to=Order,
+                              on_delete=models.CASCADE)
+    row = models.IntegerField()
+    seat = models.IntegerField()
+
+    class Meta:
+        constraints = [
+            UniqueConstraint(
+                fields=["row", "seat", "movie_session"],
+                name="unique_ticket"
+            )
+        ]
+
+    def __str__(self) -> str:
+        return f"{self.movie_session} (row: {self.row}, seat: {self.seat})"
+
+    def clean(self) -> None:
+        if self.row < 1 or self.row > self.movie_session.cinema_hall.rows:
+            raise ValidationError({
+                "row": [
+                    f"row number must be in available range: (1, rows): (1, "
+                    f"{self.movie_session.cinema_hall.rows})"
+                ]
+            })
+
+        if (self.seat < 1
+                or self.seat > self.movie_session.cinema_hall.seats_in_row):
+            raise ValidationError({
+                "seat": [
+                    f"seat number must be in "
+                    f"available range: (1, seats_in_row): (1, "
+                    f"{self.movie_session.cinema_hall.seats_in_row})"
+                ]
+            })
```

Comment:
The `clean` method in the `Ticket` model correctly validates the `row` and `seat` fields against the `CinemaHall` capacity. Ensure that this validation logic aligns with your application's requirements and that it is tested to prevent invalid data entry.